proto/hummock.proto

```diff
@@ -272,6 +274,11 @@ message CompactTaskAssignment {
   uint32 context_id = 2;
 }
 
+message BranchedSstInfo {
+  uint64 sst_id = 1;
+  map<uint64, uint64> ref_groups = 2;
```

Comment:
> Is the reason why we use a `map` here is because protobuf doesn't have a `set` type? If we don't need to persist this in etcd, we can maintain the info as a in-mem struct?

The value is appearance count in this group.

Reply:
> > Is the reason why we use a `map` here is because protobuf doesn't have a `set` type? If we don't need to persist this in etcd, we can maintain the info as a in-mem struct?
> 
> The value is appearance count in this group.

Will the same SST appear more than once in a group?

Reply:
> > > Is the reason why we use a `map` here is because protobuf doesn't have a `set` type? If we don't need to persist this in etcd, we can maintain the info as a in-mem struct?
> > 
> > 
> > The value is appearance count in this group.
> 
> Will the same SST appear more than once in a group?

after split-merge :-(

Reply:
Let's not worry about merge at this stage. Also, now that we don't persist BranchedSstInfo in etcd, it is unneccessary to use protobuf for it.

Reply:
> Let's not worry about merge at this stage. Also, now that we don't persist BranchedSstInfo in etcd, it is unneccessary to use protobuf for it.

fixed and now it is set